code/script.js

```diff
@@ -30,18 +49,177 @@ const showMessage = (message, sender) => {
+   } else if (indexDoggos === 3) {
+     userReply(message)
+     setTimeout(() => giftWrapDog(message), 1000);
+   } else if (indexDoggos === 4) {
+     console.log()
+     userReply(message)
+     setTimeout(() => nameofDog(message), 2000);           
+} else {  
+    userReply(message);
+    setTimeout(() => goodbye(message), 2000);
+}
+}
+ 
 // Starts here
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  indexDoggos = 1;
+  botReply('Hello there! Are you here for a doggo friend?🐶')
+  //the yes/no buttons here are in the index.html file (we decided not to start with a form)
+  yesButton
+    .addEventListener('click', () => nextQuestion('Hells yes!')) 
+  noButton
+    .addEventListener('click', () => reloadBot())
+  } 
+ 
+//This is the 2nd Question Asked after greeting
+const dogSize = (message) => {
+    indexDoggos++
+    botReply('🐾PAW-some!🐾 What size doggo friend would you like?');
+       
+      inputWrapper.innerHTML = ` 
+        <button id="small-btn">Small</button>
+        <button id="medium-btn">Medium</button>
+        <button id="large-btn">Large</button> 
+      `  
+    document
+      .getElementById('small-btn')
+      .addEventListener('click', () => nextQuestion('Small'))
+    document
+      .getElementById('medium-btn')
+      .addEventListener('click', () => nextQuestion('Medium'))
+    document
+      .getElementById('large-btn')
+      .addEventListener('click', () => nextQuestion('Large'))
+  }
+
+  //This is Question 3
+  const typeOfDog = (type) => {
+    indexDoggos++
+    botReply(`Excellent! A ${type} doggo! Based on that please choose a breed below!`);
+    //Here the user can select a breed from a dropdown menu based on their answer from Question 2
+    if (type === 'Small') {
+      inputWrapper.innerHTML = `
+        <select id='select'>
+          <option value="">Select Here</option>
+          <option value="Yorkshire Terrier">Yorkshire Terrier</option>
+          <option value="Chihuahua">Chihauhua</option>
+          <option value="Beagle">Beagle</option>
+          <option value="Pomeranian">Pomeranian</option>
+          <option value="Welsh Cardigan/Pembroke Corgi">Welsh Cardigan or Pembroke Corgi (Yes there 2 diff. breeds)</option>
+          <option value="Dachshund the sausage doggo">Dachshund aka Sausage dog!</option>
+        </select>`
+    } else if (type === 'Medium'){
+      inputWrapper.innerHTML = `
+        <select id='select'>
+          <option value="">Select Here</option>
+          <option value="Soft-Coated Wheaton Terrier">Soft-Coated Wheaton Terrier</option>
+          <option value="Siberian Husky">Siberian husky</option>
+          <option value="Basset Hound">Basset Hound</option>
+          <option value="Poodle">Poodle</option>
+          <option value="Golden Retriever">Golden Retriever</option>
+          <option value="Bull Terrier">Bull Terrier</option>
+        </select>`
```

Comment:
This structure is repeated a few times. Maybe you can try to create a template with the usage of `${}` structure to make it more DRY 😸